CHANGELOG.md

```diff
@@ -1,6 +1,57 @@
+* Support CHAINID, EXTCODEHASH, and SELFBALANCE instructions [#1644](https://github.com/trailofbits/manticore/pull/1644)
+* **[Changed API]** Renamed several arguments in EVM API, including `gaslimit` --> `gas` [#1652](https://github.com/trailofbits/manticore/pull/1652)
+* Explore states that self-destruct [#1699](https://github.com/trailofbits/manticore/pull/1699)
+* Lazy solving for the Ethereum leak detector [#1727](https://github.com/trailofbits/manticore/pull/1727)
+
+### Native
+* Support for ARM modified-immediate encodings [#1638](https://github.com/trailofbits/manticore/pull/1638)
+* Support for `/proc/self/maps` [#1639](https://github.com/trailofbits/manticore/pull/1639)
+* Support for `llseek` [#1640](https://github.com/trailofbits/manticore/pull/1640)
+* Support for `arm_fadvise64_64` [#1648](https://github.com/trailofbits/manticore/pull/1648)
+* Allow symbolic sockets in `accept` [#1618](https://github.com/trailofbits/manticore/pull/1618)
+* Fixes to `open` [#1657](https://github.com/trailofbits/manticore/pull/1657)
+* Overhauled filesystem emulation [#1673](https://github.com/trailofbits/manticore/pull/1673)
+* Fixed system call argument concretization [#1697](https://github.com/trailofbits/manticore/pull/1697)
+* **[Added API]** Add a symbolic model for `strcpy` [#1681](https://github.com/trailofbits/manticore/pull/1681)
+
+### WASM
+* Delay branch condition concretization for better coverage [#1641](https://github.com/trailofbits/manticore/pull/1641)
+
+### Other
+* **[Added API]** Added a snapshot system [#1710](https://github.com/trailofbits/manticore/pull/1710)
+* Transparent compression for state files [#1624](https://github.com/trailofbits/manticore/pull/1624)
+* Unify around singleton interface for solver [#1649](https://github.com/trailofbits/manticore/pull/1649)
+* Use `__slots__` to reduce memory usage in expression system [#1635](https://github.com/trailofbits/manticore/pull/1635)
+* **[Removed API]** Removed `policy` argument from ManticoreBase, added `outputspace_url` to optionally separate working files from output files [#1651](https://github.com/trailofbits/manticore/pull/1651)
+* Disable broken `get_related` logic [#1674](https://github.com/trailofbits/manticore/pull/1674)
+* Disable flaky Z3 tactics [#1691](https://github.com/trailofbits/manticore/pull/1691)
+* Remove Keystone engine from dependencies [#1684](https://github.com/trailofbits/manticore/pull/1684)
+* Improved error messages [#1632](https://github.com/trailofbits/manticore/pull/1632), [#1704](https://github.com/trailofbits/manticore/pull/1704)
+* Made ConstraintSets hashable [#1703](https://github.com/trailofbits/manticore/pull/1703)
+* Added system to dynamically enable/disable plugins [#1696](https://github.com/trailofbits/manticore/pull/1696) [#1708](https://github.com/trailofbits/manticore/pull/1708)
+* Re-establish support for alternative SMT solvers [#1714](https://github.com/trailofbits/manticore/pull/1714)
```

Comment:
If you do a commit. May be mention Yices and CVC4 ?

Reply:
Needed to update the dates anyway. Done!